crates/iota/src/client_commands.rs

```diff
@@ -342,10 +329,8 @@ pub enum IotaClientCommands {
         opts: Opts,
     },
     /// Run a PTB from the provided args
-    #[command(name = "ptb")]
```

Comment:
I hope clap is so smart to not make it `p-t-b` 😅

Reply:
Yes, I tried that